be/CMakeLists.txt

```
@@ -97,6 +97,8 @@ option(USE_AVX512 "Build with AVX512f/AVX512BW instruction" OFF)
 
 option(WITH_TENANN "Build with TENANN" OFF)
 
+option(WITH_RELATIVE_PATH "Build .c source file with relative path" OFF)
```

```suggestion
option(WITH_RELATIVE_SRC_PATH "Build source file with relative path" ON)
```

make it `ON` by default.

---

Fixed.